socialhome/views.py

```diff
@@ -49,3 +57,23 @@ def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs.update({"user": self.request.user})
         return kwargs
+
+
+class ObtainSocialhomeAuthToken(ObtainAuthToken, APIView):
+    # Documenting the API
+    schema = AutoSchema(manual_fields=[
+        coreapi.Field("username", description="User's username", required=True, location="form",
+                      schema=coreschema.String()),
+        coreapi.Field("password", description="User's password", required=True, location="form",
+                      schema=coreschema.String()),
+    ])
+
+    def post(self, request, *args, **kwargs):
+        serializer = self.serializer_class(data=request.data, context={"request": request})
+        serializer.is_valid(raise_exception=True)
+        user = serializer.validated_data["user"]
+        token, created = Token.objects.get_or_create(user=user)
+        data = LimitedProfileSerializer(user.profile, context={"request": self.request}).data
+        data.update({"token": token.key})
+        print(data)
```

Comment:
Print should be removed.

Reply:
Ah, missed this one after finishing to debug. Nice catch
.